### PR TITLE
specifications: progressive delivery

### PR DESCRIPTION
**Pull request by sergiusens**
Signed-off-by: Sergio Schvezov <sergio.schvezov@canonical.com>

- [ ] Have you followed the [guidelines for contributing](https://github.com/snapcore/snapcraft/blob/master/CONTRIBUTING.md)?
- [ ] Have you signed the [CLA](http://www.ubuntu.com/legal/contributors/)?
- [ ] Have you successfully run `./runtests.sh static`?
- [ ] Have you successfully run `./runtests.sh tests/unit`?

-----
Split out from #2916

### CODE REVIEW

File: specifications/20200329-progressive-delivery.org [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+Track     Arch    Channel    Version    Revision    Progress
+latest    all     stable     -          -           -
+                  candidate  -          -           -
+                  beta       0.6        8           23 → 20%
+                             10         13          71 → 80%
+                  edge       ↑          ↑           -
+#+END_SRC
+
+If a /progressive release/ is done on =edge= while =edge= is currently
+following =beta=, then following information shall be observed:
```
Comment (cjp256) [inline specifications/20200329-progressive-delivery.org]: I think your examples are backwards.

File: specifications/20200329-progressive-delivery.org [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+** Showing Progress
+Progress shall be displayed when used when the definitions in [[*Enabling the feature][Enabling the feature]]
+are satisfied.
+
+When run, for the case of a snap that has a /progressive release/ on the =edge=
+channel set to 80% with 71% for the /progressive release/ and 23% of the
+population on the actual release, the following behavior shall be observed:
+
+#+BEGIN_SRC sh
+$ snapcraft staging servers> snapcraft status candycane
```
Comment (cjp256) [inline specifications/20200329-progressive-delivery.org]: what's `snapcraft staging servers>`?

Comment (sergiusens) [reply specifications/20200329-progressive-delivery.org]: oops

File: specifications/20200329-progressive-delivery.org (hunk 1 above)
Comment (sergiusens) [reply specifications/20200329-progressive-delivery.org]: How so, this is exactly a verbatim copy of what we discussed in that meeting in Frankfurt

Comment (sergiusens) [reply specifications/20200329-progressive-delivery.org]: oic

Comment (sergiusens) [reply specifications/20200329-progressive-delivery.org]: when I did the oic I was looking at the table above, the example below is correct. Revision 11 is progressively released on edge, there is no current release on edge.

**Pull request merged**